app/models/merge_request_organisation.rb

```diff
@@ -0,0 +1,31 @@
+class MergeRequestOrganisation < ApplicationRecord
+  belongs_to :merge_request, class_name: "MergeRequest"
+  belongs_to :merging_organisation, class_name: "Organisation"
+  validates :merge_request_id, presence: { message: I18n.t("validations.merge_request.merge_request_id.blank") }
```

Comment:
why do we choose to validate the column `merge_request_id` as opposed to the presence of the association `merge_request_id`?

Reply:
No reason, good spot!